src/Media.js

```diff
@@ -31,6 +32,7 @@ const Media = (props) => {
     middle,
     object,
     right,
+    src,
```

Comment:
we keep `src` in the `attributes` since we access it from there below and so that it automatically gets added to the rendered element since we spread `attributes`

Reply:
So should I delete the top or bottom one xD

Reply:
So hier exactly do you need me to add src to the Media component, so that I can use it in typescript?

Reply:
none of these files control typescript, that is all handled in a separate repository by other people.

Reply:
Oh that's right , sorry for that